src/thorlabs_cube/aqctl_thorlabs_cube.py

```diff
@@ -72,12 +76,16 @@ def main():
                 dev = Tpz(args.device)
                 loop.run_until_complete(dev.get_tpz_io_settings())
             elif product == "kdc101":
-                dev = Kdc(args.device)
+
+                dev = Kdc(loop, args.device)
+            elif product == "kpz101":
+                dev = Kpz(loop, args.device)
```

Comment:
```suggestion
                dev = Kdc(args.device)
            elif product == "kpz101":
                dev = Kpz(args.device)
```
The event loop is no longer a required parameter for T/K-Cube control classes after #25.